.travis.yml

```diff
@@ -11,18 +11,15 @@ matrix:
       dist: xenial
       sudo: true
```

Comment:
I think it should be safe not to use the default image for python 3.7, you can maybe try and see if it passes

Reply:
You can also add 3.8 and remove 3.5

Reply:
Done in: 8495b32929666e122dba71e5a3e9b7ffc0c3fcd0